config/overlays/e2e-release-0.4/ironic.env

```diff
@@ -0,0 +1,10 @@
+HTTP_PORT=6180
```

Comment:
The name of the directory could be ironic-release-0.4 or e2e-ironic-release-0.4, otherwise having e2e- in one and fixture- in another seems a bit out of convention.

Reply:
The name of these directories are to aligned with the equivalent from `main`: `fixtures` and `e2e`. I think your naming convention makes senses, but do you think we should change these original names (`fixtures` and `e2e`) as well? What about you @lentzi90 ?

Reply:
I don't have a strong preference. It looks alright like this to me.
My reasoning when I named the folders `e2e` and `fixture` is that they are used for the e2e and fixture tests respectively. So I think it also makes sense then with e2e-release-04 and fixture-release-0.4

Reply:
Ok. I dont have a strong opinion either. This is certainly not a blocker for the tests to go in.

/approve